docs/UserGuide.md

```diff
@@ -219,13 +219,32 @@ A success message will be displayed if the supplier is successfully deleted.
 
 ### Mark a supplier with a status : `mark -s`
 
-The `mark` command is used to mark a supplier as either **active** or **inactive**
-in VendorVault. This helps you keep track of which suppliers are currently active for deliveries and which are not.
+The `mark` command is used to mark a supplier as either **active** or **inactive**.
+in Vendor Vault. This helps you keep track of which suppliers are currently active for deliveries and which are not.
+
+<box type="tip" seamless>
+
+**Tip:**
+- A supplier has a default status of `active`.
+- A supplier with an `inactive` status signifies that the supplier is not currently active for deliveries. 
+However, marking an `active` supplier as `inactive` will not delete the delivery associated with the supplier from the delivery list 
+or affect the delivery status in any way.
+</box>
 
 Format: `mark -s SUPPLIER_INDEX STATUS`
-- `SUPPLIER_INDEX`: Must be a number greater than 0 and must not be blank.
+- `INDEX`: Must be a number greater than 0 and must not be blank.
 - `STATUS`: Must be one of the following: `active`, `inactive` and must not be blank.
 
+<box type="warning" seamless>
+
+**Warnings**:
+- A spacing between `mark` and `-s` is compulsory.
+- A spacing between `-s` and `INDEX` is compulsory.
```

Comment:
Should this be SUPPLIER_INDEX

Reply:
but it's shown as INDEX in the error message in the app itself